graph/clients/listenNoteClient.js

```diff
@@ -0,0 +1,32 @@
+import unirest from 'unirest'
+
+const BASE_URL = 'https://listen-api.listennotes.com/api/v2'
+
+const callListenNote=async (endpoint)=>{
+  const response = await unirest.get(`${BASE_URL}${endpoint}`)
+    .header('X-ListenAPI-Key', process.env.LISTEN_NOTE_API_KEY)
+  return response.body
+}
+
+export const getBestPodcasts = async (page = 1, region='us', safe_mode=1)=>{
+  return callListenNote(`/best_podcasts?page=${page}&region=${region}&safe_mode=${safe_mode}`)
+}
+
+export const getPodcast = async (podcast_id)=>{
+  return callListenNote(`/podcasts/${podcast_id}?&sort=recent_first`)
+}
+
+export const getEpisode = async (episode_id)=>{
+  return callListenNote(`/episodes/${episode_id}`)
+}
+
+export const search = async (query, sort_by_date=0, type='podcast', offset=0, len_min=10, len_max=100,
```

Comment:
This function takes in quite a number of parameters, let pass in an object instead